### PR TITLE
Sqa gray case fix

### PR DESCRIPTION
**Pull request by Jagadeesh-ajjada**
- SQA unanswerable questions updated to void gray cases.
- Removed unnecessary characters from the user input to achieve optimal results in the confusion matrix.
- Also removed some of the python code standard warnings like mentioned below.
image

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -34,35 +37,40 @@ def __init__(self, session_state=st.session_state):
...
+            "What are the available free software for a student",
+            "Where are all the printers located",
+            "What are the CoyoteLabs virtual computer labs",
+            "Is Adobe Creative Cloud available as student software",
+            "What is information security awareness",
+            "How do I enable multi-factor authentication",
+            "What are Coyote OneCard benefits",
+            "What if i lost my campus laptop charger",
         }
         self.answerable_questions = {q.lower() for q in self.answerable_questions}
```
Comment (smrchanda877) [inline app.py:51]: Can you change this to use the `remove_special_characters` instead of `.lower()`?

File: app.py [hunk 2]
```diff
@@ -34,35 +37,40 @@ def __init__(self, session_state=st.session_state):
...
+            "Can you write code for a basic Python script",
+            "What is the CGI phone number or email",
+            "What class does Dr. Alzahrani teach",
+            "Who is Hironori Washizaki",
+            "How can I make a payment for the tuition fee",
+            "What is the future impact of AI on software quality standards",
+            "What is regression testing",
+            "How much does parking cost for one semester",
         }
         self.unanswerable_questions = {q.lower() for q in self.unanswerable_questions}
```
Comment (smrchanda877) [inline app.py:64]: Can you change this to use the `remove_special_characters` instead of `.lower()`?

**Review by smrchanda877 — CHANGES_REQUESTED** [review]
@Jagadeesh-ajjada Can you make it so that clicking on Confusion matrix text on UI takes the user to read me SQA questions. Right now, it takes the user to the SQA questions in the wiki.

Also, can you change the wiki to update the new SQA questions?

**Comment by Jagadeesh-ajjada** [comment]
Instead of looping to remove elements from the RAW text, we could directly rewrite the RAW text to eliminate the need for two loops. What do you think?

**Review by smrchanda877 — APPROVED** [review]
LGTM

**Pull request merged**